implementations/rust/ockam/ockam_command/src/node/start.rs

```diff
@@ -74,7 +74,7 @@ async fn run_impl(
         None,                                          // Credential
         None,                                          // Trust Context
         None,                                          // Project Name
-        node_setup.disable_file_logging,
+        true,
```

Comment:
Can we please add a trailing comment like the above lines had, since the bare boolean is a little less clear about intent or content?

Reply:
Done!